src/simulate.py

```diff
@@ -395,7 +439,22 @@ def matcher_func(n1, n2):
                         if func in ["Buf", "MainMem"]:
                             scaling = node_data["size"]
                             logger.info(f"latency scaling: {scaling}")
-
+                        
+                        # wire latency
+                        # can't really do a workaround here because the path is built on the computation_dfg, and that means 
+                        # I would have to find a substitute for the node for the whole path, so instead we will just ignore for now
+                        # print("wire latency")
+                        node_data["in other graph"] = "allocation" in node_data and node_data["allocation"] != "" and "Mem" not in node_data["allocation"] and "Buf" not in node_data["allocation"]
+                        node_index = path.index(node)
+                        if node_index != 0:
+                            node_index_prev = node_index - 1
+                            node_data_prev = computation_dfg.nodes[path[node_index_prev]]
+                            # print("({}, {})".format(node_data_prev["allocation"], node_data["allocation"]))
+                            if node_data["in other graph"] and node_data_prev["in other graph"] and hw.parasitic_graph.has_edge(node_data_prev["allocation"], node_data["allocation"]):
+                                print("true")
+                                parasitic_edge = hw.parasitic_graph[node_data_prev["allocation"]][node_data["allocation"]]
+                                net_delay = parasitic_edge["net_cap"] * parasitic_edge["net_res"] * 1e-3 # pico -> nano
```

Comment:
each parasitic edge should have a list of Rs and Cs right. what happened to that? 
And here it should be R1C1 + (R1+R2)C2 ...

Reply:
some edges are not lists

Reply:
since the computation graph is wonky, it messes with what edges are used. The edges that are looked at don't have res and cap lists, so i just ignored it. I can implement the R1C1 + (R1+R2)C2 tho.